core/neat/settings/_settings.scss

```diff
@@ -11,7 +11,19 @@
 ///
 /// @access private
 
-$neat-default-grid: (
+$_neat-grid-default: (
```

Comment:
Should this be `-defaults` (with an “s”) since it houses multiple default settings?